### DISCUSSION TITLE
2023 Edition

### DISCUSSION THREAD

**Discussion by MichaelCade** [MichaelCade/90DaysOfDevOps#236] [Announcements]
We will very soon be announcing the plan for the 2023 edition

**Comment by thisisshivanshpatel** [comment]
Really excited for this 🙂

**Reply by MichaelCade** [reply]
Thanks. What are we missing or what should be updated.

**Comment by MichaelCade** [comment]
We are half way through the 2023 edition, How are people finding it? Are people waiting till the end to take it all in at once? Also it is not too early to discuss the areas and topics we are missing

**Reply by abhishekkmukherjee** [reply]
I've started the 2022 edition today so will not be able to start 2023 now

**Comment by vigopina** [comment]
Hi I wish to unsubscribe from this email thread please help me do that

**Reply by MichaelCade** [reply]
It will be based on your notification settings on GitHub this should help https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/managing-subscriptions-for-activity-on-github/managing-your-subscriptions

**Comment by ParaZzzedgar** [comment]
HI. This project so cool!
